SpaceScrapper/Assets/_Game/_Scripts/Runtime/Global/Game.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SpaceScrapper
+{
+    /// <summary>
+    /// Backbone of the Game
+    /// </summary>
+    public static class Game
+    {
+        public static SceneContext SceneContext { get; private set; }
+        public static GameEvents Events { get; private set; }
```

Comment:
Why even go through Game.GameEvents, when we could just go directly to GamEvents tho

Reply:
Centralises where we keep the reference and allows us to safely reset it in the method which will be handy for later when we will disable domain reload on playmode enter